test/content/timeout-issue.js

```diff
@@ -0,0 +1,37 @@
+const { test } = require('tap')
+
+function cleanCache (module) {
+  delete require.cache[require.resolve(module)]
+}
+
+function requireFresh (module) {
+  cleanCache(module)
+  return require(module)
+}
+
+test('timeout issue content', async t => {
+  t.afterEach(() => {
+    delete process.env.GITHUB_HOST
+    delete process.env.GITHUB_URL
+    cleanCache('../../lib/env')
+  })
+
+  t.test('includes the link to the initial branch on the custom github host', async t => {
+    process.env.GITHUB_HOST = 'https://enterprise.github/api/v3/'
```

Comment:
It's not new, it existed before (https://github.com/greenkeeperio/greenkeeper/blob/46d0f51d54380a175ee68af3a96cf36dafa76453/lib/github.js#L36 & the test for it) ... that's why I've used it.